src/sidebar/services/groups.js

```diff
@@ -66,7 +116,11 @@ function groups($rootScope, store, api, isSidebar, localStorage, serviceUrl, ses
         params.document_uri = uri;
       }
       documentUri = uri;
-      return api.groups.list(params);
+      return api.groups.list(params, null, { includeMetadata: true });
```

Comment:
Perhaps insert a comment above this line saying that it's invoking the API? Maybe that's really obvious, but it could be helpful?

Reply:
Done